src/bsp/src/stm32/include/SpiEsp.h

```diff
@@ -47,7 +47,7 @@ class SpiEsp : public ICommInterface {
         uint16_t m_payloadSize;
     } m_inboundMessage, m_outboundMessage;
 
-    std::array<uint8_t, ESP_SPI_MAX_MESSAGE_LENGTH> m_data;
+    std::array<uint8_t, 4 * ESP_SPI_MAX_MESSAGE_LENGTH> m_data;
```

Comment:
Why 4 times the message length?

Reply:
So that we can store 4 messages of the max length while the deserializer reads them